src/components/notifications/GrafanaOncall.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+    <div class="mb-3">
+        <label for="GrafanaOncallURL" class="form-label">{{ $t("Grafana Oncall URL") }}<span style="color: red;"><sup>*</sup></span></label>
```

Comment:
This string should be added to `en.json`, otherwise our translation team will not be able to translate this

Reply:
@atmaniak you can resolve this comment by changing this to
```suggestion
        <label for="GrafanaOncallURL" class="form-label">{{ $t("GrafanaOncallURL") }}<span style="color: red;"><sup>*</sup></span></label>
```